src/Windows/R/Components/Impl/InteractiveWorkflow/Commands/SessionInformationCommand.cs

```diff
@@ -88,16 +88,21 @@ public sealed class SessionInformationCommand : IAsyncCommand {
             sb.AppendLine("\t" + Resources.PhysicalMemory.FormatInvariant(aboutHost.TotalPhysicalMemory, aboutHost.FreePhysicalMemory));
             sb.AppendLine("\t" + Resources.VirtualMemory.FormatInvariant(aboutHost.TotalVirtualMemory, aboutHost.FreeVirtualMemory));
 
-            if (!string.IsNullOrEmpty(aboutHost.VideoCardName)) {
-                sb.AppendLine("\t" + Resources.VideoCardName.FormatInvariant(aboutHost.VideoCardName));
-            }
+            int count = 1;
+            foreach(var cardInfo in aboutHost.VideoCards) {
```

Comment:
That won't skip a value if it is not present.

---
In reply to: [115848894](https://github.com/Microsoft/RTVS/pull/3516#discussion_r115848894) [](ancestors = 115848894)